extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/pom.xml

```diff
@@ -90,6 +90,12 @@
             <artifactId>quarkus-jaxrs-client-reactive-deployment</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-architecture-tests</artifactId>
+            <type>test-jar</type>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Better also add the classifier, see https://maven.apache.org/plugins/maven-jar-plugin/examples/create-test-jar.html
But I think this will be obsolete as soon as this lives under `test-framework`, which would also match "The preferred way" from the above article.

Reply:
It seems like the "preferred way" doesn't work with `dependenciesToScan`

Reply:
How so? Is is expecting type `test-jar` and/or classifier `tests`? (on my phone ATM)

Reply:
The tests in the jar where not being run when I used the preferred way